docs/source/user_guide_rag.rst

```diff
@@ -393,12 +393,13 @@ into a vector is required. Therefore, the retriever requires knowledge of an emb
 Embedders
 -----------------------------
 
-Currently, this package supports several embedders:
-- `OpenAIEmbeddings`
-- `AzureOpenAIEmbeddings`
-- `VertexAIEmbeddings`
-- `CohereEmbeddings`
-- `SentenceTransformerEmbeddings`.
+Currently, this package supports the following embedders:
+
+- :ref:`openaiembeddings`
+- :ref:`sentencetransformerembeddings`
+- :ref:`vertexaiembeddings`
+- :ref:`mistralaiembeddings`
```

Comment:
Nice to have links here! Do we need to add `AzureOpenAIEmbeddings` as well?

Reply:
https://github.com/neo4j/neo4j-graphrag-python/pull/142/commits/fef874886d880cc886e39af1b0464f20bc9250b8